BrainPortal/app/controllers/signups_controller.rb

```diff
@@ -266,6 +276,34 @@ def resend_conf_multi #:nodoc:
     render :action => :multi_action
   end
 
+  def hide_multi #:nodoc:
+    reqids = params[:reqids] || []
+    reqs   = Signup.find(reqids)
+
+    reqs.each do |req|
+      req[:hidden] = true
+      req.save
+    end
+
+    flash[:notice] = "Hid " + view_pluralize(reqs.size, "record")
+
+    redirect_to :action => :index
+  end
+
+  def show_multi #:nodoc:
```

Comment:
Not sure: how will the admin be able to select the records to SHOW if they are already hidden?!?